### ISSUE TITLE
processflow 2.0.2 hanging on both anvil/blues and rhea

### ISSUE THREAD

**Issue by xylar** [E3SM-Project/processflow#117]
On both systems, I'm attempting to run `processflow.py -v`.  In both cases, the command hangs in a way that I'm not able to interrupt with `^C`.  Eventually, I have to kill the process.

On blues, I attempt to run 
```
source /lcrc/soft/climate/e3sm-unified/base/etc/profile.d/conda.sh
conda activate e3sm_unified_1.2.1_py2.7_nox
unset LD_LIBRARY_PATH
processflow.py -v
``` 
and on rhea:
```
module use /ccs/proj/cli900/sw/rhea/modulefiles/all
module load e3sm-unified/1.2.1
processflow.py -v
```

**Comment by xylar** [comment]
This is a duplicate of https://github.com/E3SM-Project/e3sm-unified/issues/22

**Comment by sterlingbaldwin** [comment]
The task Im currently working on is to make sure the processflow works without any hitches on all the supported machines. 2.0.2 actually had a couple bugs that I didnt realize were there until after I published it. Im going to rebuild and publish 2.0.3 hopefully today, which will include a PBS interface to work with the resource manager at LCRC. 

I havent used rhea, so Im not sure what sort of complications stem from it. Could you email me the processflow.log, your config, and any console output?

**Comment by xylar** [comment]
@sterlingbaldwin, please feel free to try the 3 commands I listed above.  

For `rhea`, there is no processflow.log produced.  There is not config file -- I'm not really trying to run processflow, just make sure the script will tell me its version.  There is literally *no* console output whatever.  The process just hangs and cannot even be killed with `^C`.  Again, please try it out yourself.  For now, I will not be testing `processflow` on blues/anvil or rhea because it blocks the rest of the e3sm-unified installation process.

**Comment by xylar** [comment]
Okay, sounds good.  As long as these fixes are in the next release of processflow, I'll just re-enable my checks on all machines when I install the next e3sm-unified.  I'll keep an eye out for when the next version is released.  (I discovered that it's not safe to assume that the latest version in the e3sm channel is a release version.)

**Comment by xylar** [comment]
Resolved in the latest e3sm-unified (1.2.2)
